soc/nxp/rw/Kconfig

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022-2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RW6XX
+	select ARM
+	select CPU_CORTEX_M33
+	select CPU_CORTEX_M_HAS_DWT
+	select CLOCK_CONTROL
+	select PLATFORM_SPECIFIC_INIT
+	select CPU_HAS_ARM_SAU
+	select CPU_HAS_ARM_MPU
+	select CPU_HAS_FPU
+	select HAS_MCUX_OS_TIMER
+	select ARM_TRUSTZONE_M
+	select CPU_CORTEX_M_HAS_SYSTICK
+	select HAS_MCUX
+	select HAS_MCUX_FLEXCOMM
+	select INIT_SYS_PLL
+	select HAS_MCUX_CACHE
+	select HAS_PM
```

Comment:
Don't believe we should select this- the SOC port here does not have PM, so this will cause a build failure on PM samples